setup.cfg

```diff
@@ -16,6 +16,7 @@ classifier =
         Programming Language :: Python :: 3.5
         Programming Language :: Python :: 3.6
         Programming Language :: Python :: 3.7
+        Programming Language :: Python :: 3.8
```

Comment:
I tend to leave off the classifiers for minor Python versions to avoid the need for this kind of change, and use the `python_requires` metadata to specify a minimum supported version. This is a matter of preference, though.

Reply:
IMHO, it needs changing every 12 months or so, it's not such a hassle and advertise that it's tested on python 3.8 as well. The `python_requires` should always be used and that's not the case for `auditwheel`. I pushed another commit fixing this.